src/operator/src/error.rs

```diff
@@ -28,6 +28,9 @@ use snafu::{Location, Snafu};
 #[snafu(visibility(pub))]
 #[stack_trace_debug]
 pub enum Error {
+    #[snafu(display("Table already exists: `{}`", table))]
+    TableAlreadyExist { table: String, location: Location },
```

Comment:
```suggestion
    TableAlreadyExists { table: String, location: Location },
```

Reply:
I think it’s better to use `TableAlreadyExist`?

Reply:
The error message is `already exists`.